pkg/statistics/handle/globalstats/global_stats.go

```diff
@@ -381,7 +381,7 @@ func WriteGlobalStatsToStorage(statsHandle statstypes.StatsHandle, globalStats *
 			util.StatsMetaHistorySourceAnalyze,
 		)
 		if err != nil {
-			statslogutil.StatsLogger().Error("save global-level stats to storage failed",
+			statslogutil.StatsLogger().Warn("save global-level stats to storage failed",
```

Comment:
wrapped by the foreground analyze's error or the error log of `auto analyze failed`.